### ISSUE TITLE
[TW-829] Include annotations when running start command

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#864]
_Max Muller on 2010-11-29T20:12:09Z says:_ 

 I love the new feature that allows timestamps to be added as annotations to tasks.

Here's a related feature request: Allow the start command to include annotation text along with the time stamp. 

I often use the start command to mark a task as active. I often then annotate it to include a date stamped record of what happened. So, for example, if I have the task:

>    1    Ask boss for a raise. 

I would first type: "start 1" to mark the task as active,
And then type "annotate 1" to include a note about what happened that got the task started.

>    1  *  Ask boss for a raise. 
>          11/22/10 Sent a email to request a meeting time

It would speed things up a little if users could simply include the annotation text after the start command, like this "start 1 Sent email to request a meeting time" 

This would be similar to the annotations with the "wait" command--which would allows users to indicate WHO exactly they are waiting for--which I saw proposed elsewhere on this site.

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2010-11-29T20:12:09Z
Modified: 2014-02-09T02:06:39Z
```

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2011-01-09T15:32:54Z says:_ 

 Reasonable request.  Needs new parser.

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2011-07-10T05:58:08Z says:_ 

 This was fixed while addressing feature #422 earlier.  You can now do:

<pre>
task start 1 Sent email to request a meeting time
</pre>

... just as suggested.  Thanks Max.
